config/routes.rb

```diff
@@ -1,10 +1,23 @@
 Rails.application.routes.draw do
   devise_for :users
+
+  namespace :api do
+    namespace :v1 do
+      resources :users, only: [:index, :show] do
+        resources :posts, only: [:index, :show] do
+          resources :comments, only: [:index, :create, :new, :destroy]
+          resources :likes, only: [:create, :new]
+        end
+      end
+    end
+  end
+
+  
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍